### PR TITLE
"D" Spells Level 2

### PR DESCRIPTION
**Pull request by ChrisSpangler**
Completed 'D" Spells Level 2

### CODE REVIEW

File: _posts/spells/second-level/2020-10-15-spells-2-dragons-breath.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Dragon's Breath
+permalink: "/spells/second-level/dragons-breath/"
+layout: spell-post
+categories: [Spells, Second Level]
+tags: [Sorcerer, Wizard]
+source: "Xanathar's Guide to Everything (XGtE)"
+spell-level: 2nd-Level
+school: Conjuration
```
Comment (magicalmusings) [inline _posts/spells/second-level/2020-10-15-spells-2-dragons-breath.md]: Transmutation

**Review by magicalmusings — CHANGES_REQUESTED** [review]
Just the quick change on one file and it's good for merge! Glad I have you helping me out with this @ChrisSpangler

**Comment by ChrisSpangler** [comment]
fixed

**Review by magicalmusings — APPROVED** [review]

**Comment by magicalmusings** [comment]
Looks good on my end, thank you for your help. I'll merge shortly

**Pull request merged**